scripts/submit_hf.py

```diff
@@ -34,7 +34,7 @@ def gen_command_template(params, machine, seed=const.HF_DEFAULT_SEED):
         "fd_statlist": params.FD_STATLIST,
         "hf_bin_path": sim_struct.get_hf_bin_path(params.sim_dir),
         HF_VEL_MOD_1D: params["hf"][HF_VEL_MOD_1D],
-        "duration": params.sim_duration,
```

Comment:
Is there a scenario where we want sim_duration=0?
Seems like this should be changed to normal dictionary syntax if anything

Reply:
X( Of course this went on the remove diff, not the add diff